app/assets/javascripts/templates/directives/markdown_input.jst.ejs

```diff
@@ -0,0 +1,6 @@
+<div class="text-box">
+  <textarea class="atwho" ng-model="content" placeholder="{{ filler }}"></textarea>
```

Comment:
is the ng-model always going to be `content`? or does that need to be customisable?